### PR TITLE
Proof of concept script for kubernetes images

### PR DESCRIPTION
**Pull request by afbjorklund**
Issue #1922

### CODE REVIEW

**Comment by jandubois** [comment]
> In case you are wondering, [nörd](https://sv.wikipedia.org/wiki/N%C3%B6rd) is the Swedish spelling. 🇸🇪

Cute, but a bit awkward to type even with tab completion (unless you use a scandinavian/german keyboard layout), e.g. on English keyboard: `n` `⌥u` `o`, `⇥`. Using tab completion on `n` alone gives too many alternatives...

File: cmd/nördctl.lima
Comment (AkihiroSuda) [inline cmd/nördctl.lima:1]: Probably this should be under the `contrib/`dir or a third party repo

**Comment by afbjorklund** [comment]
I did not find a good name at the time, so it was a work in progress

File: cmd/nördctl.lima
Comment (afbjorklund) [reply cmd/nördctl.lima:1]: That probably just means that it will not be available to Podman Desktop

Comment (afbjorklund) [reply cmd/nördctl.lima:1]: You mean something like `kulima`, right?

**Pull request closed without merging**